File: End-Sem/Slot 1/Q1.c
#include <stdio.h>

void main()
{
	int num1, num2, num3;

	// Taking user input for 3 numbers
	printf("Enter number 1: ");
	scanf("%d", &num1);
	printf("Enter number 2: ");
	scanf("%d", &num2);
	printf("Enter number 3: ");
	scanf("%d", &num3);
	printf("\n");

	// Calculating average
	float average = (float)(num1 + num2 + num3) / 3;
	printf("Average: %f\n", average);
}

File: Semester_4/CS2201 (Intro. to Computation)/CT 2/23MS233/Q1.py
#Taking list input from user
l = eval(input("Enter a list of numbers: "))
s = 0

#Checking if the total of 2 numbers of the list is equal to s and the 2 numbers are not equal
for i in range(len(l)):
    for j in range(i, len(l)):
        
        if (l[i] + l[j] == s) and (l[i] != l[j]):
            print(f"{i}, {j}")